tests/base.py

```diff
@@ -212,11 +212,33 @@ def wrapper(self, url, params=NoDefault, **kw):
 
         return wrapper
 
+    def bulk_json_api_method(method):
+        def wrapper(self, url, params=NoDefault, **kw):
```

Comment:
There's a lot of duplication with `json_api_method` here. Can you refactor this? Perhaps use a helper method to generate the `wrapper` method, i.e. `make_wrapper(self, method, content_type, name) -> wrapped method)`